### ISSUE TITLE
Can't download a release - Virus Detected

### ISSUE THREAD

**Issue by rlevchenko** [doitintl/kube-no-trouble#178]
I tried to download the latest preview release https://github.com/doitintl/kube-no-trouble/releases/tag/nightly-0.4.0-83-g055df4e and it was interrupted by windows antivirus: Trojan:Win32/Wacatac.B!ml found. Are there any reasons for that?

**Comment by stepanstipl** [comment]
HI @rlevchenko, thanks for bringing this to our attention 👍 . This is likely due to our binaries being compresses by [UPX](https://upx.github.io/), which is a great tool, but due to the nature of its functionality, this might be one of the drawbacks. I've checked and there seems to be a [few known issues with false positives](https://github.com/upx/upx/search?q=virus&type=issues).

I'm not sure what we can do about this, some options:
- Look into adding scanning for viruses as part of our release process (sound like a good idea anyway, opened #179)
- Removing UPX - we have considered this previously, as originally the binaries using k8s go client were huge [over 100mB], this is a bit better now... maybe?
- Providing another set of binaries without UPX - might solve the issue, but I'm not fan of the duplicity/additional effort

TL;DR: I'm confident there's no reason for concern and this is a false positive due to UPX use.

**Comment by rlevchenko** [comment]
@stepanstipl Thanks for response. 1) AV scanning as a part of the CD - great idea 2) I'm not an expert in this area, may be other exe packers available in the market that not marked as malware. I assume tests are needed with diff. set of tools 

Anyway, I downloaded the release (just allowed the file on the device in the AV settings), works great. 

Thanks for your work!
